app/(stateless-toolbar)/submit-url/page.tsx

```diff
@@ -49,7 +49,7 @@ const SubmitUrl = () => {
 
   const [creatingTag, setCreatingTag] = useState<boolean>(false)
   const [description, setDescription] = useState<string>("")
-  const [duplicateTagError, setDuplicateTagError] = useState<string>("")
+  const [duplicateTagError, setDuplicateTagError] = useState<boolean>(false)
```

Comment:
not sure why you're changing these to boolean but if you are change the variable names to lower confusion for other devs. `hasDuplicateError` or `isDuplicateError`.

This could remain a string and pass boolean checks though *shrug*

Reply:
Resolved in https://github.com/State-Channel-4/frontend/pull/43/commits/7db0975c5ae7dcec7e8df0682120044cbd32394e